coremltools/converters/mil/frontend/torch/ops.py

```diff
@@ -4486,6 +4560,15 @@ def expand_as(context, node):
     context.add(res)
 
 
+def _arange(context, node_name, start, end, step):
```

Comment:
nit: type hint?

Reply:
Added type hints except for `context`, since `from .converter import TranscriptionContext` would cause circular import...

Reply:
You can always do the "string type hint"